questions.md

```diff
@@ -0,0 +1,57 @@
+Why are all server data retrieval requests as POST?  They should be GET
```

Comment:
Do not make a questions.md file like this, there's too large a risk of it making it into the repo when someone merges a request. nothing should be in a PR that is not intended for merge. If you have questions send them to me in email/slack, or add them to the Comment you create when you create the PR in the first place. 

Remove this file from the PR.